README.md

```diff
@@ -4,10 +4,11 @@ Python support for Sublime's LSP plugin provided through [microsoft/pyright](htt
 
 ## Installation
 
-1. Install [LSP](https://packagecontrol.io/packages/LSP) and
+1. Make sure you have Node.js installed and `node` is in your `$PATH`. The language server subprocess is a Node.js app.
```

Comment:
As per `plugin.py`, the minimum required version of node seems to be `12.0.0`. Maybe worth adding it to avoid ambiguity.